en/docs/chapter_searching/searching_algorithm_revisited.md

```diff
@@ -1,48 +1,48 @@
 
 ## Brute-force search
 
-Brute-force search locates the target element by traversing every element of the data structure.
+A Brute-force search locates the target element by traversing every element of the data structure.
 
-- "Linear search" is suitable for linear data structures such as arrays and linked lists. It starts from one end of the data structure, accesses each element one by one, until the target element is found or the other end is reached without finding the target element.
-- "Breadth-first search" and "Depth-first search" are two traversal strategies for graphs and trees. Breadth-first search starts from the initial node and searches layer by layer, accessing nodes from near to far. Depth-first search starts from the initial node, follows a path until the end, then backtracks and tries other paths until the entire data structure is traversed.
+- "Linear search" is suitable for linear data structures such as arrays and linked lists. It starts from one end of the data structure and accesses each element one by one until the target element is found or the other end is reached without finding the target element.
+- "Breadth-first search" and "Depth-first search" are two traversal strategies for graphs and trees. Breadth-first search starts from the initial node and searches layer by layer (left to right), accessing nodes from near to far. Depth-first search starts from the initial node, follows a path until the end (top to bottom), then backtracks and tries other paths until the entire data structure is traversed.
 
-The advantage of brute-force search is its simplicity and versatility, **no need for data preprocessing and the help of additional data structures**.
+The advantage of brute-force search is its simplicity and versatility, **no need for data preprocessing or the help of additional data structures**.
 
-However, **the time complexity of this type of algorithm is $O(n)$**, where $n$ is the number of elements, so the performance is poor in cases of large data volumes.
+However, **the time complexity of this type of algorithm is $O(n)$**, where $n$ is the number of elements, so the performance is poor with large data sets.
 
 ## Adaptive search
 
-Adaptive search uses the unique properties of data (such as order) to optimize the search process, thereby locating the target element more efficiently.
+An Adaptive search uses the unique properties of data (such as order) to optimize the search process, thereby locating the target element more efficiently.
 
 - "Binary search" uses the orderliness of data to achieve efficient searching, only suitable for arrays.
 - "Hash search" uses a hash table to establish a key-value mapping between search data and target data, thus implementing the query operation.
 - "Tree search" in a specific tree structure (such as a binary search tree), quickly eliminates nodes based on node value comparisons, thus locating the target element.
```

Comment:
I think the last part "which leads to better performance" is unnecessary and the two sentences before it (line 
 27&28) should be changed as well to maintain consistency. Here is the context for your reference:

自适应搜索利用数据的特有属性（例如有序性）来优化搜索过程，从而更高效地定位目标元素。

“二分查找”利用数据的有序性实现高效查找，仅适用于数组。
“哈希查找”利用哈希表将搜索数据和目标数据建立为键值对映射，从而实现查询操作。
“树查找”在特定的树结构（例如二叉搜索树）中，基于比较节点值来快速排除节点，从而定位目标元素。

此类算法的优点是效率高 ......

You can see that the explanation focuses on how different search methods locate the target element, rather than on their performance. Therefore, performance is not the focus, and the other two sentences do not mention it either.